interactions/models/internal/tasks/task.py

```diff
@@ -75,25 +75,25 @@ def on_error(self, error: Exception) -> None:
         self.on_error_sentry_hook(error)
         interactions.Client.default_error_handler("Task", error)
 
-    async def __call__(self) -> None:
+    async def __call__(self, *args, **kwargs) -> None:
         try:
             if inspect.iscoroutinefunction(self.callback):
-                val = await self.callback()
+                val = await self.callback(*args, **kwargs)
             else:
-                val = self.callback()
+                val = self.callback(*args, **kwargs)
 
             if isinstance(val, BaseTrigger):
                 self.reschedule(val)
         except Exception as e:
             self.on_error(e)
 
-    def _fire(self, fire_time: datetime) -> None:
+    def _fire(self, fire_time: datetime, *args, **kwargs) -> None:
         """Called when the task is being fired."""
-        self.trigger.set_last_call_time(fire_time)
```

Comment:
Why is this changed?

Reply:
i just couldn't traceback to set_last_call_time. might be an issue on my end but the quickest fix i found was just to use last_call_time

Reply:
Suggesting this was added in another PR to fix something (#1438), I'm not sure if this is really the right solution.

Reply:
yeah, you're definitely right. seems i missed that pr (oops) so ill change it back. thanks for telling me!